### ISSUE TITLE
catalog_init error on managed PostgreSQL database (Oracle Cloud)

### ISSUE THREAD

**Issue by micgerber** [zebbra/data_aggregator#452]
Hi @hwuethrich,

With your [fix](https://github.com/zebbra/data_aggregator/pull/449) the migration run smoothly.
Then I run [catalog_init](https://github.com/zebbra/data_aggregator/blob/main/docs/deployment.md#roll-out) and I got this error : 

```
** (Ash.Error.Unknown) Unknown Error

* ** (ArgumentError) cannot load `"infofauna"` as type #Ash.Type.Atom.EctoType<[]> for field :center in #DataAggregator.Taxonomy.Catalogs.SwissSpecies<__meta__: #Ecto.Schema.Metadata<:loaded, "swiss_species">, id: nil, taxon_id_ch: nil, accepted_name: nil, usage_key: nil, accepted_usage_key: nil, scientific_name: nil, rank: nil, center: nil, inserted_at: nil, updated_at: nil, aggregates: %{}, calculations: %{}, ...>
  (ecto 3.12.3) lib/ecto/repo/queryable.ex:431: Ecto.Repo.Queryable.struct_load!/6
  (ecto 3.12.3) lib/ecto/repo/queryable.ex:246: anonymous fn/5 in Ecto.Repo.Queryable.preprocessor/3
  (elixir 1.17.2) lib/enum.ex:1703: Enum."-map/2-lists^map/1-1-"/2
  (ecto 3.12.3) lib/ecto/repo/queryable.ex:237: Ecto.Repo.Queryable.execute/4
  (ecto 3.12.3) lib/ecto/repo/queryable.ex:19: Ecto.Repo.Queryable.all/3
  (ash_postgres 2.3.1) lib/data_layer.ex:769: anonymous fn/3 in AshPostgres.DataLayer.run_query/2
  (ash_postgres 2.3.1) lib/data_layer.ex:767: AshPostgres.DataLayer.run_query/2
  (ash 3.4.8) lib/ash/actions/read/read.ex:2600: Ash.Actions.Read.run_query/4
    (elixir 1.17.2) lib/process.ex:864: Process.info/2
    (ash 3.4.8) lib/ash/error/unknown.ex:3: Ash.Error.Unknown."exception (overridable 2)"/1
    (ash 3.4.8) /app/deps/splode/lib/splode.ex:211: Ash.Error.to_class/2
    (ash 3.4.8) lib/ash/error/error.ex:66: Ash.Error.to_error_class/2
    (ash 3.4.8) lib/ash/actions/read/read.ex:351: anonymous fn/3 in Ash.Actions.Read.do_run/3
    (ash 3.4.8) lib/ash/actions/read/read.ex:280: Ash.Actions.Read.do_run/3
    (ash 3.4.8) lib/ash/actions/read/read.ex:81: anonymous fn/3 in Ash.Actions.Read.run/3
```

With the previous image (e820429ca562) I was using, the script worked.

Tanks for your help.

**Comment by sidler92** [comment]
Hey @micgerber 

There should be a new docker Image available when you repull latest.
I attempted a fix for This issue and #456.

Let me know if it works or if its still failing.

Cheers

**Comment by micgerber** [comment]
Hi @sidler92, 

Thanks for the fix.
catalog_init runs without problem now.

I still have an error while running users_init.
I'll report it under #456.
